test-utils/src/lib.rs

```diff
@@ -88,6 +88,12 @@ pub fn delta_path_for_version(version: u64, suffix: &str) -> Path {
     Path::from(path.as_str())
 }
 
+// get an ObjectStore path for a checkpoint file, based on version, part number, and total number of parts
+pub fn delta_path_for_multipart_checkpoint(version: u64, part_num: u64, num_parts: u64) -> Path {
+    let path =
+        format!("_delta_log/{version:020}.checkpoint.{part_num:010}.{num_parts:010}.parquet");
+    Path::from(path.as_str())
+}
```

Comment:
not sure if this should go here or just in some test-only code right now? can always pull it out if we need it here?